R/app_server.R

```diff
@@ -223,6 +223,38 @@ app_server <- function(input, output, session) {
     get_comm_data(selected_pkg$name())
   })
   
+  loaded2_db <- eventReactive(selected_pkg$name(), {
+    req(selected_pkg$name())
+    req(selected_pkg$name() != "-")
+    
+    dbSelect("SELECT name, version, score FROM package")
+  })
```

Comment:
I think this value should be triggered when changes to the database occur, not on package selection. Also, it would make sense to use `session$userData` instead of passing it as an argument.

Reply:
OK using session$userData now.